--- haproxy-2.9.9.orig/Makefile
+++ haproxy-2.9.9/Makefile
@@ -169,6 +169,7 @@
 #### Toolchain options.
 CC = cc
 LD = $(CC)
+PKG_CONFIG ?= pkg-config
 
 #### Default optimizations
 # Those are integrated early in the list of CFLAGS, and may be overridden by
--- haproxy-2.9.9.orig/admin/wireshark-dissectors/peers/Makefile
+++ haproxy-2.9.9/admin/wireshark-dissectors/peers/Makefile
@@ -1,5 +1,5 @@
-CFLAGS  = `pkg-config --cflags wireshark` -g -fPIC $(OPTS)
-LDFLAGS = `pkg-config --libs wireshark`
+CFLAGS  = `$(PKG_CONFIG) --cflags wireshark` -g -fPIC $(OPTS)
+LDFLAGS = `$(PKG_CONFIG) --libs wireshark`
 
 NAME = packet-happp.so
 OBJS = packet-happp.o
